src/app/core/import/import-column-mapping/import-column-mapping.service.spec.ts

```diff
@@ -0,0 +1,129 @@
+import { TestBed } from "@angular/core/testing";
+
+import { ImportColumnMappingService } from "./import-column-mapping.service";
+import { ColumnMapping } from "../column-mapping";
+import { Child } from "app/child-dev-project/children/model/child";
+import { School } from "app/child-dev-project/schools/model/school";
+import { EntitySchema } from "app/core/entity/schema/entity-schema";
+
+describe("ImportColumnMappingService", () => {
+  let service: ImportColumnMappingService;
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({});
+    service = TestBed.inject(ImportColumnMappingService);
+  });
+
+  it("should be created", () => {
+    expect(service).toBeTruthy();
+  });
+
+  it("should map columns to their respective property names automatically", () => {
+    // setup
+    const testMapping: ColumnMapping[] = [
+      { column: "name" },
+      { column: "phone" },
+      { column: "address" },
+      { column: "some other remarks" },
+    ];
+    const entitySchema: EntitySchema = new Map([
+      ["name", {}],
+      ["phone", {}],
+      ["address", {}],
+      ["some other remarks", {}],
+    ]);
+
+    // run
+    service.automaticallySelectMappings(testMapping, entitySchema);
+
+    // check
+    expect(testMapping).toEqual([
+      { column: "name", propertyName: "name" },
+      { column: "phone", propertyName: "phone" },
+      { column: "address", propertyName: "address" },
+      { column: "some other remarks" },
+    ]);
+  });
```

Comment:
each `it(...)` block is one test case, so it's enough if you duplicate this, rather than having a second `fdescribe(...)` block.

You can see that there are already two test cases within the fdescribe: `it("should be created" ...)` and `it("should map columns to their respective property names automatically" ...)`. Add your new test as a third it() block and clean up this file so it only has a single fdescribe().